### PR TITLE
#79 change application serial to string

### PR DESCRIPTION
**Pull request by nmadruga**
Required for Front-end PR of issue application-manager-web-app PR #31 -#33 (https://github.com/openmsupply/application-manager-web-app/pull/50)

### CODE REVIEW

File: database/buildSchema/16_application.sql [hunk 1]
```diff
@@ -6,10 +6,9 @@ CREATE TYPE public.trigger as ENUM ('onApplicationCreate', 'onApplicationSubmit'
 
 CREATE TABLE public.application (
     id serial primary key,
-    unique_identifier varchar,
     template_id integer references public.template(id),
     user_id integer references public.user(id),
-    serial integer,
+    serial varchar,
```
Comment (CarlosNZ) [inline database/buildSchema/16_application.sql:11]:
```suggestion
serial varchar UNIQUE,
```

**Review by CarlosNZ — CHANGES_REQUESTED** [review]
Maybe constrain field to being UNIQUE?

**Comment by nmadruga** [comment]
> Maybe constrain field to being UNIQUE?

Yes. that's better. Thanks

**Review by CarlosNZ — APPROVED** [review]
👍

**Pull request merged**